specification/resources/resource-manager/Microsoft.Resources/stable/2021-04-01/resources.json

```diff
@@ -4605,6 +4608,50 @@
       ],
       "description": "Deployment properties."
     },
+    "DeploymentParameter": {
+      "properties": {
+        "value": {
+          "type": "object",
```

Comment:
Should/can it be something more specific than just `object`?

Reply:
You also need to keep in mind that most property values could be an ARM expression (which is a string in JSON, but there's a defined type for this somewhere), so be careful not to make expressions cause a validation error in any of these changes.

Reply:
So something like this:

```json
"value": { }
```

I'm coming from .NET where `object` is the least specific type you can think of.

**Question**: would keeping `object` hurt anything down the road? ARM templates, SDK, etc.

Reply:
Have we closed on having just this:

```json
"parameters": {
  "type": "object",
  "additionalProperties": {
    "$ref": "#/definitions/DeploymentParameter"
  },
  "description": "..."
},
```

i.e. what it's currently?

Reply:
I think the last comment definitely makes sense to me. However, the `value` property in `DeploymentParameter` is still of object type. We do need to allow any value there, so you should probably model it as an empty schema (or schema with just a description) like an earlier comment in this discussion.